practice.js

```diff
@@ -1,14 +1,26 @@
+//Importing all the common nodes needed throughout the code
 let input = document.getElementById("input");
 let div = document.getElementById("result");
+let add = document.getElementById("add");
+let clearButton = document.getElementById('clear-button');
 
+//Function to handle disable attribute of add button
+const addChangeHandler = (e) => {
+    e ? add.removeAttribute('disabled') : add.setAttribute('disabled', 'true');
```

Comment:
just checking e's value will be reliable?

Reply:
Yes, I have checked this before using it.